security/libressl-devel/Portfile

```diff
@@ -23,9 +23,8 @@ homepage            https://www.libressl.org
 conflicts           openssl libressl
 
 master_sites        openbsd:LibreSSL
-checksums           rmd160  e2fff57fe8ae432a4b3104561d2ba1ee16804242 \
-                    sha256  1e3a9fada06c1c060011470ad0ff960de28f9a0515277d7336f7e09362517da6 \
-                    size    3359012
+checksums           rmd160 92cbbe323ae16197e8ae90fb8e7e5d783910b7d0 \
+                    sha256 af2bba965b06063518eec6f192d411631dfe1d07713760c67c3c29d348789dc3
```

Comment:
Why remove the `size` checksum?

Reply:
@janstary If you want us to commit things reasonably quickly, you should perhaps reply to minor issues like this reasonably quickly?

Reply:
The fact that the file has the givensha256 checksum and the given rmd160 checksum
makes it almost certain (as certain as crypto checksums can be) that it is the intended file.
The fact that it has the prescribed size adds nothing to that certainty.
Why do you want to have it there?

Reply:
@janstary Because it's in the default checksums list: https://github.com/macports/macports-base/commit/95a153e6a4bff8b99502806960c916a2b5d20504.

Reply:
You can argue on the developer's list about whether it's sensible or not, but it is current practice. If you want that practice changed, argue there, not on an individual PR submission.

Reply:
Anyway, please re-add the "size" field.

Reply:
@janstary Being unwilling to add a single field seems pretty self-defeating.